src/main/java/com/example/gradecalculator/web/model/WebSecurityConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.example.gradecalculator.web.model;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityCustomizer;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
```

Comment:
could also be moved into configuration folder

Reply:
done